adapters/sovrn/sovrn.go

```diff
@@ -20,6 +20,11 @@ type SovrnAdapter struct {
 	URI string
 }
 
+type SovrnImpExt struct {
```

Comment:
scope of `SovrnImpExt` is within   `sovrn` package. Therefore this struct can be private

```
- type SovrnImpExt struct {
+ type sovrnImpExt struct {    //   's' is lowercase here

```

Reply:
Nice catch, thank you. Fixed.